internal/querynode/query_collection.go

```diff
@@ -942,22 +963,26 @@ func (q *queryCollection) search(msg queryMsg) error {
 	hisSearchResults, sealedSegmentSearched, err1 := q.historical.search(searchRequests, collection.id, searchMsg.PartitionIDs, plan, travelTimestamp)
 	if err1 != nil {
 		log.Warn(err1.Error())
-		return err1
```

Comment:
Why not return error？Even if there is no dm data in historical, search shouldn't return error.